dkg/types.go

```diff
@@ -36,4 +36,5 @@ type Config struct {
 	Storage             Storage
 	SignatureDomainType types.DomainType
 	Signer              types.DKGSigner
+	Verifier            types.DKGVerifier
```

Comment:
why do we need it here? it should be in the validateSignedOutput func

Reply:
It is used in `validateSignedOutput`. It's included in the config so that we can use a different verification strategy (i.e. mock) for testing. Otherwise, we won't be able to use the ecdsa in spectest as ecdsa involves random number and hence non-deterministic.

Reply:
You verify that the signature is valid, that should be ok in spec tests as long as the operators have the same eth address.
I'm for removing it, all those types of validations should be constant